databases/week2/Part3.sql

```diff
@@ -0,0 +1,182 @@
+insert into user_task (user_id, task_id) values(3, 19);
+insert into user_task (user_id, task_id) values(4, 24);
+insert into user_task (user_id, task_id) values(4, 20);
+insert into user_task (user_id, task_id) values(5, 20);
+insert into user_task (user_id, task_id) values(5, 18);
+insert into user_task (user_id, task_id) values(5, 15);
+insert into user_task (user_id, task_id) values(6, 10);
+insert into user_task (user_id, task_id) values(6, 7);
+insert into user_task (user_id, task_id) values(6, 27);
+insert into user_task (user_id, task_id) values(7, 33);
+insert into user_task (user_id, task_id) values(7, 18);
+insert into user_task (user_id, task_id) values(7, 23);
+insert into user_task (user_id, task_id) values(8, 26);
+insert into user_task (user_id, task_id) values(8, 30);
+insert into user_task (user_id, task_id) values(8, 11);
+insert into user_task (user_id, task_id) values(9, 34);
+insert into user_task (user_id, task_id) values(9, 15);
+insert into user_task (user_id, task_id) values(9, 1);
+insert into user_task (user_id, task_id) values(10, 29);
+insert into user_task (user_id, task_id) values(10, 16);
+insert into user_task (user_id, task_id) values(10, 1);
+insert into user_task (user_id, task_id) values(11, 26);
+insert into user_task (user_id, task_id) values(11, 27);
+insert into user_task (user_id, task_id) values(11, 17);
+insert into user_task (user_id, task_id) values(11, 2);
+insert into user_task (user_id, task_id) values(1, 3);
+insert into user_task (user_id, task_id) values(2, 6);
+insert into user_task (user_id, task_id) values(3, 8);
+insert into user_task (user_id, task_id) values(4, 9);
+insert into user_task (user_id, task_id) values(5, 12);
+insert into user_task (user_id, task_id) values(6, 14);
+insert into user_task (user_id, task_id) values(7, 21);
+insert into user_task (user_id, task_id) values(8, 25);
+insert into user_task (user_id, task_id) values(9, 28);
+insert into user_task (user_id, task_id) values(10, 31);
+insert into user_task (user_id, task_id) values(11, 32);
+
+
+/*Get all the tasks assigned to users whose email ends in @spotify.com*/
+
+SELECT task.title, task.description , user.name, user.email
+FROM user_task 
+JOIN user ON user_task.user_id=user.id
+JOIN task ON task.id=user_task.task_id
+WHERE user.email LIKE '%@spotify.com';
+
+/*Get all the tasks for 'Donald Duck' with status 'Not started'*/
+
+SELECT task.title, task.description, user.name,status.name
+FROM user_task
+JOIN user ON user_task.user_id=user.id
+JOIN task ON task.id=user_task.task_id
+JOIN status ON status.id=task.status_id
+WHERE user.name="Donald Duck" AND status.name="Not started";
+
+/*Get all the tasks for 'Maryrose Meadows' that were created in september (hint: month(created)=month_number)*/
+
+SELECT task.title, task.description, user.name, task.created
+FROM user_task
+JOIN user ON user_task.user_id=user.id
+JOIN task ON task.id=user_task.task_id
+WHERE user.name="Donald Duck" AND MONTH(task.created)="09";
```

Comment:
Here you have to change "Donald Duck" to "Maryrose Meadows", so your query can fit the subject. By the way your query is good